plugins/lookup/collection_version.py

```diff
@@ -0,0 +1,129 @@
+# (c) 2021, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+name: collection_version
+author: Felix Fontein (@felixfontein)
+version_added: "4.0.0"
+short_description: Retrieves the version of an installed collection
+description:
+  - This lookup allows to query the version of an installed collection, and to determine whether a
+    collection is installed at all.
+  - By default it returns C(none) for non-existing collections and C(*) for collections without a
+    version number. The latter should only happen in development environments, or when installing
+    a collection from git which has no version in its C(galaxy.yml). This behavior can be adjusted
+    by providing other values with I(result_not_found) and I(result_no_version).
+options:
+  _terms:
+    description:
+      - The collections to look for.
+      - For example C(community.general).
+    type: list
+    elements: str
+    required: true
+  result_not_found:
```

Comment:
Is this parameter missing a default value field here?

Reply:
Not really, since Python `None` is the default. I'll add it, but likely it will get ignored anyway.

Reply:
I am asking this because usually ` None` means "user did not supply this argument and is not set at all". I know in this case the end result is the same, but from the documentation perspective, I would expect to see the default value here. That being said, I do not know if test plugins are documentable yet, so ... ;)

Reply:
Fixed in e0d7058a3c7b825b55ad2bb5130d1471ee06b322.

Reply:
It's a lookup btw, not a test plugin :) But the `None` won't show up in the HTML docs since antsibull only shows them if `value['default'] is not none`... (This is a problem with normalizing the data structures, which causes an explicit `None` to be indistinguishable from 'no default provided'.)